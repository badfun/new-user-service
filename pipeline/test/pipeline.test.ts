import Pipeline = require('../lib/pipeline-stack');
import { Stack } from '@aws-cdk/core';
import { SynthUtils, MatchStyle } from '@aws-cdk/assert';

// snapshot test
test('Pipeline stack matches template', () => {
  const stack = new Stack();

  new Pipeline.PipelineStack(stack, 'Pipeline');
  expect(SynthUtils.toCloudFormation(stack)).toMatchSnapshot();
});
