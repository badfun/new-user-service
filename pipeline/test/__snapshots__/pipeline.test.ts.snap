// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pipeline stack matches template 1`] = `Object {}`;
